ajax/libs/inferno-create-element/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "inferno-create-element",
+  "filename": "inferno-create-element.min.js",
+  "version": "3.7.0",
+  "author": {
+    "email": "dg@domgan.com",
+    "name": "Dominic Gannaway"
+  },
+  "description": "Provides methods to create Inferno VNodes",
+  "homepage": "https://infernojs.org/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/infernojs/inferno.git"
+  },
+  "keywords": [
+    "babel",
+    "framework",
+    "inferno",
+    "interfaces",
+    "react",
+    "user interfaces"
+  ],
+  "license": "MIT",
+  "npmName": "inferno-create-element",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/inferno-create-element!(*.node)*.+(js|map|css)"
```

Comment:
You should use `!(ts)` inside `dist` instead of using `+(js|map|css)`, that's a basic convention right?

Reply:
cc @cdnjs/intern3

Reply:
Yes, that's right. Will also pay attention to similar folders such as build and release.